src/components/FocusTrap/FocusTrapForScreen/index.web.tsx

```diff
@@ -51,12 +52,13 @@ function FocusTrapForScreen({children}: FocusTrapProps) {
                 trapStack: sharedTrapStack,
                 allowOutsideClick: true,
                 fallbackFocus: document.body,
-                // We don't want to ovverride autofocus on these screens.
-                initialFocus: () => {
-                    if (screensWithAutofocus.includes(activeRouteName)) {
+                delayInitialFocus: CONST.ANIMATED_TRANSITION,
+                // We don’t want to override autofocus while there is a focused element.
+                initialFocus: (containers) => {
+                    const hasFocusedElement = containers?.some((container) => container.contains(document.activeElement));
+                    if (hasFocusedElement) {
                         return false;
                     }
-                    return undefined;
                 },
                 setReturnFocus: (element) => {
```

Comment:
Remove `setReturnFocus`

Reply:
It's used to return the focus to the element that opened the modal, e.g. if you press button1 that opens a RHP, after closing the RHP, button1 should be focused again. We should follow the same principle here, that is to refocus button1 as long as we don't have another focused element.